chanzuckerberg/corpora-data-portal
# thuang-HOTFIX-ToS-links

This HOTFIX updates the ToS and Privacy links to match the new ones and will merge directly into `prod` branch:

E.g.,

OLD:
`/static/deploy/*`

NEW:
`/static/cellxgene/deploy/*`

Proof:
<img width="594" alt="Screen Shot 2020-08-31 at 3 46 35 PM" src="https://user-images.githubusercontent.com/6309723/91776022-74334300-eba1-11ea-8724-d727b1a22040.png">
<img width="626" alt="Screen Shot 2020-08-31 at 3 46 40 PM" src="https://user-images.githubusercontent.com/6309723/91776029-75fd0680-eba1-11ea-9569-b303cce4a8a9.png">

QA Steps:
1. Run `npm run develop`
2. Load `http://localhost:8000/`
3. Hover over ToS and Privacy links to make sure it matches the new pattern

Thank you!

**NOTE: After prod deployment, we'll need to back merge `prod` into `main`, so `main` gets the fix too!**
